ermrest/sql/upgrade_registry.sql

```diff
@@ -16,3 +16,26 @@
 --
 
 DROP INDEX IF EXISTS ermrest.simple_registry_id_deleted_on_idx;
+
+DO $$
+BEGIN
+
+IF (SELECT True
+    FROM information_schema.columns
+    WHERE table_schema = 'ermrest'
+      AND table_name = 'simple_registry'
+      AND column_name = 'alias_target') IS NULL THEN
+
+  -- perform catalog naming feature upgrade
+  ALTER TABLE ermrest.simple_registry
```

Comment:
so the `descriptor` doesn't have to be up upgraded to `jsonb` format?

Reply:
Thanks, that was an oversight and should be fixed now...